tox.ini

```diff
@@ -3,6 +3,7 @@ envlist = py27, py35, py36, py37, linters
 
 [travis]
 python =
+    3.8: py38
```

Comment:
This adds support for Python 3.8. Please update `setup.py` as well to ensure this is publicly visible.

Reply:
👍